### PR TITLE
Suggest the correct update command in global mode

### PR DESCRIPTION
**Pull request by canac**
Updating global packages requires running `npm install --global` rather than `npm install --save[-dev]`.

### CODE REVIEW

**Comment by LinusU** [comment]
This will break on Node.js 0.12 which we support. Can you refactor this to not use template strings?

**Comment by canac** [comment]
@LinusU Oh, well I already switched it to string concatenation. Do you want me to switch it back to template strings?

**Comment by LinusU** [comment]
Sorry, I jumped the gun a bit on that comment. Template string is fine to use since we are using Babel.js 😁 

How about moving the flags out though?

```js
const flags = currentState.get('global') ? '--global' : `--save${pkg.devDependency ? '-dev' : ''}`
```

**Comment by canac** [comment]
OK, I was trying to change as little as possible, but I can do that refactoring to reduce redundancy.

**Comment by LinusU** [comment]
Thank you 🙌

**Comment by canac** [comment]
I finished with that refactor.

File: lib/out/static-output.js [hunk 1]
```diff
@@ -9,23 +9,26 @@ function uppercaseFirstLetter(str) {
...
 
-function render(pkg) {
+function render(pkg, currentState) {
     const packageName = pkg.moduleName;
     const rows = [];
 
     const indent = '           ' + emoji('   ');
+    const flags = currentState.get('global') ? '--global' : `--save${pkg.devDependency ? '-dev' : ''}`;
+    const upgradeCommand = `npm install ${flags} ${packageName}@${pkg.latest}`;
+    const updateMessage = `${chalk.green(upgradeCommand)} to go from ${pkg.installed} to ${pkg.latest}`;
```
Comment (LinusU) [inline lib/out/static-output.js]: small nit but should it be `upgradeMessage`? To align with `upgradeCommand` and `easyUpgrade`...

**Comment by LinusU** [comment]
Nice 👍 

I'll merge as soon as the CI is green

**Comment by canac** [comment]
@LinusU Is this ready to be merged now?

**Comment by LinusU** [comment]
Sorry for the delay, I have been traveling with limited internet access. Merging now, thanks for the help!

**Comment by LinusU** [comment]
Released as 5.2.3

**Pull request merged**